docs/src/views/Colors.tsx

```diff
@@ -0,0 +1,352 @@
+        },
+        Text: {
+          root: ({ theme: { siteVariables } }) => ({
+            '& a': {
+              color: siteVariables.colorScheme.brand.foreground,
+            },
+          }),
+        },
+      },
+      componentVariables: {
+        ColorBox: colorBoxVariables,
+      },
+    }}
+  >
+    <ProviderConsumer
+      render={({
+        siteVariables: { colors, contextualColors, naturalColors, transparentColors },
+      }) => (
+        <DocPage title="Colors">
+          <Header as="h2">Introduction</Header>
+          <p>
+            The organizing of the colors for an application has many requirements and constraints.
+            There is a need to be intentional and functional with color use. We analyzed existing
+            frameworks and picked the best ideas from them. In Stardust, the colors mechanisms are
+            completely based on the <code>siteVariables</code>, we don't have any additional API
+            specific to the colors.
+          </p>
+          <p>
+            There are two things important in order for colors to work transparently when the theme
+            switching is in play:
+          </p>
+          <ul>
+            <li>
+              <Text weight="bold" color="primary">
+                {link('Color palette', '#color-palette')}
+              </Text>{' '}
+              - central place for all colors available in the application,
+            </li>
+            <li>
+              <Text weight="bold" color="primary">
+                {link('Color scheme', '#color-scheme')}
+              </Text>{' '}
+              - design tokens for all colors used in the application that should be appropriately
+              mapped in all themes.
+            </li>
+          </ul>
+          <p>
+            Be aware that everything that follows is our recommendation, not the requirement
+            (everything will work even if you'll decide to structure the palette and schemas
+            differently in your theme).
+          </p>
+          <Header as="h2" content="Color palette" />
+          <p>
+            Colors in {link('Teams color palette', '/color-palette')} have the following
+            categorization.
+          </p>
+
+          <Header as="h3">Primitive colors</Header>
+          <p>
+            This part of the palette contains colors that, semantically, cannot have any tints. This
+            group is represented by two colors, {code('black')} and {code('white')} - as there is
+            nothing blacker than black and nothing whiter than white.
+          </p>
+
+          <Grid columns={2}>
+            {_.map(['black', 'white'], color => (
+              <div key={color}>
+                <ColorBox
+                  name={color}
+                  rounded
+                  size="big"
+                  value={colors[color]}
+                  copyToClipboardIcon={false}
+                />
+              </div>
+            ))}
+          </Grid>
```

Comment:
### padding: 4rem

![image](https://user-images.githubusercontent.com/14183168/59513348-f6f7b180-8eba-11e9-9de5-6b136ee1bbdb.png)

### padding: .75rem

![image](https://user-images.githubusercontent.com/14183168/59513319-e6dfd200-8eba-11e9-969a-0ba5232a85a1.png)

I vote for second

Reply:
All Headers for the color are the same, would not like to add special case for these two colors.

Reply:
Will make all of them smaller. 👍